tests/lib/snaps/basic-iface-hooks-producer/meta/snap.yaml

```diff
@@ -1,8 +1,7 @@
 name: basic-iface-hooks-producer
 version: 1.0
 slots:
-    bar:
-        interface: content
-        content: abc
-        read:
-            - $SNAP/etc
\ No newline at end of file
+    producer:
+        interface: test-snapd
+        producer-attr1: producervalue1
```

Comment:
Could you please use `producer-value-1` (unless you feel strongly about it here), the key could be called `producer-attr-1`.